graph/auxiliary/abis/Anchor.json

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "constant": true,
+    "inputs": [],
+    "name": "organization",
+    "outputs": [{ "name": "", "type": "address" }],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "constant": true,
+    "inputs": [],
+    "name": "coAnchor",
+    "outputs": [{ "name": "", "type": "address" }],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "inputs": [
+      { "name": "_remoteChainId", "type": "uint256" },
+      { "name": "_blockHeight", "type": "uint256" },
+      { "name": "_stateRoot", "type": "bytes32" },
+      { "name": "_maxStateRoots", "type": "uint256" },
+      { "name": "_organization", "type": "address" }
+    ],
+    "payable": false,
+    "stateMutability": "nonpayable",
+    "type": "constructor"
+  },
+  {
+    "anonymous": false,
+    "inputs": [
+      { "indexed": false, "name": "_blockHeight", "type": "uint256" },
+      { "indexed": false, "name": "_stateRoot", "type": "bytes32" }
+    ],
+    "name": "StateRootAvailable",
+    "type": "event",
+    "_alias": "StateRootAvailable"
+  },
+  {
+    "constant": false,
+    "inputs": [{ "name": "_coAnchor", "type": "address" }],
+    "name": "setCoAnchorAddress",
+    "outputs": [{ "name": "success_", "type": "bool" }],
+    "payable": false,
+    "stateMutability": "nonpayable",
+    "type": "function"
+  },
+  {
+    "constant": true,
+    "inputs": [{ "name": "_blockHeight", "type": "uint256" }],
+    "name": "getStateRoot",
+    "outputs": [{ "name": "stateRoot_", "type": "bytes32" }],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "constant": true,
+    "inputs": [],
+    "name": "getLatestStateRootBlockHeight",
+    "outputs": [{ "name": "height_", "type": "uint256" }],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "constant": false,
+    "inputs": [
+      { "name": "_blockHeight", "type": "uint256" },
+      { "name": "_stateRoot", "type": "bytes32" }
+    ],
+    "name": "anchorStateRoot",
+    "outputs": [{ "name": "success_", "type": "bool" }],
+    "payable": false,
+    "stateMutability": "nonpayable",
+    "type": "function"
+  },
+  {
+    "constant": true,
+    "inputs": [],
+    "name": "getRemoteChainId",
+    "outputs": [{ "name": "remoteChainId_", "type": "uint256" }],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  }
+]
```

Comment:
Do we need the abis here again 🤔 ❓ 
We already populated ABIs needed for respective subgraphs as part of below ticket implementation. We can give path e.g. abi/0.10.0/Anchor.json in the subgraph.yml. I compared both the ABIs and they are same. This way we can avoid duplication.
https://github.com/OpenST/mosaic-chains/issues/85
https://github.com/OpenST/mosaic-chains/pull/88/files

Reply:
I think yes, It's kind of enforcement from the graph while running `codegen` and `deploy-local`. It expects ABI's to be kept into a specific folder which is written to IPFS node on deployment.

Reply:
ok. In that case I think we should delete the directory `abi/0.10.0` which we populated for this ticket.
Also Anchor.json is missing from graph/origin/subgraph.yaml.mustache.

Reply:
What do you mean my missing? For current scope, origin chain doesn't need Anchor subgraph.

Also, We can't delete `abi/0.10.0` because to run `graph init` abis are needed. Which is keptt in `abi/0.10.0`.  graph init then copies it to desired folder.

Reply:
As discussed, we can keep abi for now. In the ticket #87, it can be evaluated if these abis are duplicate or not.